src/Theme/index.js

```diff
@@ -0,0 +1,83 @@
+import { createGlobalStyle } from 'styled-components';
+import reset from 'styled-reset';
+import './libs';
+
+const GlobalStyle = createGlobalStyle`
+${reset}
+
+:root {
+  /* Define Colors */
+  --red: #E05433;
+  --white: #FFFFFF;
+  --grey: #ced4da;
+  --lightGrey: #f8f9fa;
+  --grey-500: #6c757d;
+  --darkGrey: #495057;
+  --black: #212529;
```

Comment:
Please add gradation to gray color, something similar to 100, 200, 300, 400

Reply:
Done